lib/active_model/serializer/attributes.rb

```diff
@@ -0,0 +1,59 @@
+module ActiveModel
+  class Serializer
+    # Defines the attributes to be serialized.
+    #
+    module Attributes
```

Comment:
Fixed [the situation](https://github.com/beauby/active_model_serializers/commit/af8fc7e80ed00f67df68ba0f5ad15efcd78d2508). Reverted to overriding `inherited` at module-level, but calling `super` each time in order to decorate the serializers.